crates/matrix-sdk-qrcode/src/types.rs

```diff
@@ -389,23 +385,25 @@ impl VerificationData {
     /// Create a new `VerificationData` struct that can be encoded as a QR code.
     ///
     /// # Arguments
-    /// * `event_id` - The event id of the `m.key.verification.request` event
-    /// that initiated the verification flow this QR code should be part of.
+    /// * `flow_id` - The event ID or transaction ID of the
+    /// `m.key.verification.request` event that initiated the
+    /// verification flow this QR code should be part of.
     ///
-    /// * `first_key` - Our own cross signing master key. Needs to be encoded as
+    /// * `first_master_key` - Our own cross signing master key. Needs to be
+    ///   encoded as
```

Comment:
Oh since we switched to the vodozemac type, this doesn't need to be unpadded base64 anymore.